modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/rel/IgniteIndexScan.java

```diff
@@ -126,7 +121,9 @@ private IgniteIndexScan(
     /** {@inheritDoc} */
     @Override protected RelWriter explainTerms0(RelWriter pw) {
         return super.explainTerms0(pw)
-            .itemIf("sourceId", sourceId, sourceId != -1);
+            .itemIf("sourceId", sourceId, sourceId != -1)
+            .item("collation", collation())
+            .item("idxCollation", idxCollation);
```

Comment:
why do we need both `collation` and `idxCollation`?

Reply:
`idxCollation` for index bounds, since `collation` an be trimmed.
I've removed `idxCollation` now, and add collation restoration by index bound.